nomad/structs/secure_variables.go

```diff
@@ -167,16 +167,22 @@ type SecureVariablesQuota struct {
 }
 
 type SecureVariablesUpsertRequest struct {
-	Data []*SecureVariableDecrypted
+	Data       []*SecureVariableDecrypted
+	CheckIndex *uint64
 	WriteRequest
 }
 
+func (svur *SecureVariablesUpsertRequest) SetCheckIndex(ci uint64) {
+	svur.CheckIndex = &ci
+}
+
 type SecureVariablesEncryptedUpsertRequest struct {
 	Data []*SecureVariableEncrypted
 	WriteRequest
 }
 
 type SecureVariablesUpsertResponse struct {
+	Conflict []*SecureVariableDecrypted
```

Comment:
Maybe pluralize this because it's a slice? Or does it really need to be a slice to begin with?

```suggestion
	Conflicts []*SecureVariableDecrypted
```